src/prerequisites.py

```diff
@@ -35,6 +35,8 @@ def fetch_remote_git_history(ref):
     print("No merge instance branch found. Exiting.")
     sys.exit(1)
 
+write_to_github_output(f"merge_instance_branch={merge_instance_branch}")
```

Comment:
This is just logging, essentially?

Reply:
This, which is called during the [prerequisite](https://github.com/trunk-io/nx-action/blob/main/action.yaml#L50) step of this workflow, is outputting to github's output, which gets passed around between all of the remaining steps in the flow. That's why we switch to passing in `steps.prerequisites.outputs.merge_instance_branch` for the merge branch for the other steps ([link](https://github.com/trunk-io/nx-action/blob/main/action.yaml#L67)). We do this because providing the merge branch to us is optional - we use the repo's default branch if it isn't provided, and we make the prerequisite step figure that out for us ([link](https://github.com/trunk-io/nx-action/blob/main/src/prerequisites.py#L30-L32))